test/unit/ripple/index.test.tsx

```diff
@@ -5,7 +5,9 @@ import {MDCRippleFoundation} from '@material/ripple/foundation';
 import {MDCRippleAdapter} from '@material/ripple/adapter';
 import {SpecificEventListener} from '@material/base/types';
 import {mount} from 'enzyme';
-import {withRipple, InjectedProps} from '../../../packages/ripple/index';
+
+import Button from '../../../packages/button';
+import {withRipple, InjectedProps} from '../../../packages/ripple';
```

Comment:
do you need this import here? It looks unused.

Reply:
Yeah this is just my habit.
I always separate node_module and local_module imports!
Does it look better to put it back??

Reply:
Maybe I misunderstood your comment.
This code already existed before.
`import {withRipple, InjectedProps} from '../../../packages/ripple';`
I'm just moved it down.

Reply:
ah! Sorry I didn't notice that. Yes that was there before. Looks good.